.github/workflows/ci.yml

```
@@ -0,0 +1,32 @@
+name: Python package
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Add the following now that you're using environment variables:
```suggestion
    runs-on: ubuntu-latest
    env:
        TOKEN: test-token
        RECAPTA_SECRET: recapta-test
```
Or maybe there's a more elegant way to handle this if the environment variable isn't set... I'm not sure.